localization/languages/pt-BR.json

```diff
@@ -194,10 +192,10 @@
         /* PDF Viewer */
         "PDFPageCounter": {"unsafeHTML": "página <input type='text'/> de <span id='total'></span>"},
         /* Download manager */
-        "downloadCancel": null, //missing translation
-        "downloadStateCompleted": null, //missing translation
-        "downloadStateFailed": null, //missing translation
+        "downloadCancel": "Download cancelado",
+        "downloadStateCompleted": "Download concluído",
+        "downloadStateFailed": "Download falhou",
         /* Update Notifications */
-        "updateNotificationTitle": null //missing translation
+        "updateNotificationTitle": "Atualizar título da notificação"
```

Comment:
"updateNotificationTitle" is just the identifier given to this (and all the other strings) to reference them in the code. The actual string (from en-US.json) is "A new version of Min is available".